packages/wrangler/src/__tests__/whoami.test.tsx

```diff
@@ -6,6 +6,8 @@ import { getUserInfo, WhoAmI } from "../whoami";
 import { mockConsoleMethods } from "./helpers/mock-console";
 import { useMockIsTTY } from "./helpers/mock-istty";
 import { msw } from "./helpers/msw";
+import { mswSucessOauthHandlers } from "./helpers/msw/handlers/oauth";
+import { mswSucessUserHandlers } from "./helpers/msw/handlers/user";
```

Comment:
We can do a barrel import file if this gets unwieldy

Reply:
This is a definitely a style thing, but what about `import { default as mswSucessOauthHandlers } from "..."`? That way each individual file only needs an `export default ...` rather than `export const mswSucessOauthHandlers ...`

Reply:
I prefer named exports and imports, forced people to actually name the thing imported the same as the thing being defined and exported.